src/strings/PowerFxResources.en-US.resx

```diff
@@ -4410,8 +4410,8 @@
     <comment>Warning for makers that a certain capability should be avoided.</comment>
   </data>
   <data name="ErrTypeLiteral_InvalidTypeDefinition" xml:space="preserve">
-    <value>Type literal declaration was invalid.</value>
-    <comment>Error message when type binding is invalid.</comment>
+    <value>Type literal declaration is invalid. The highlighted expression can not be used in Type definition.</value>
```

Comment:
>highlighted [](http://example.com/codeflow?start=52&length=11)

This is assuming that this error will be seen in a formula bar (where we can have highlighting), but that may not always be the case. It can be seen in the app checker, or in the CLI, where there are no highlighting capabilities.